core/controllers/editor.py

```diff
@@ -124,7 +124,6 @@ def get(self, exploration_id):
 
         self.values.update({
             'INTERACTION_SPECS': interaction_registry.Registry.get_all_specs(),
-            'DEFAULT_OBJECT_VALUES': obj_services.get_default_object_values(),
```

Comment:
Do we still need to import `obj_services` in this file? Ditto for the other files when you remove `obj_services.get_default_object_values()`